integration/specs/CodeInput/codeInput-1.spec.js

```diff
@@ -0,0 +1,107 @@
+const PageCodeInput = require('../../../src/components/CodeInput/pageObject');
+const { DELETE_KEY, TAB_KEY } = require('../../constants');
+
+const CODEINPUT = '#codeinput-1';
+
+describe('CodeInput base example', () => {
+    beforeAll(() => {
+        browser.url('/#!/CodeInput/1');
+    });
+    beforeEach(() => {
+        browser.refresh();
+        const component = $(CODEINPUT);
+        component.waitForExist();
+    });
+    it('should have focus on the first input on load', () => {
+        const codeInput = new PageCodeInput(CODEINPUT);
+        expect(codeInput.getFocusedIndex()).toBe(0);
+    });
+    it('should change focus to the second input when the first input its filled with a number', () => {
+        const codeInput = new PageCodeInput(CODEINPUT);
+        browser.keys(['', '2']);
+        expect(codeInput.getFocusedIndex()).toBe(1);
+    });
+    it('should remove value from first input when first input is selected, has a value and "Backspace" key is pressed', () => {
+        const codeInput = new PageCodeInput(CODEINPUT);
+        browser.keys(['', '2']);
+        expect(codeInput.getInputValueByIndex(0)).toBe('2');
+        browser.keys(DELETE_KEY);
+        expect(codeInput.getInputValueByIndex(0)).toBe('');
+    });
+    it('should return focus to the first input and clear his value when the first input has a value, the second input is empty, is focused and "Backspace" key is pressed', () => {
+        const codeInput = new PageCodeInput(CODEINPUT);
+        browser.keys(['', '2']);
+        expect(codeInput.getInputValueByIndex(0)).toBe('2');
+        expect(codeInput.getFocusedIndex()).toBe(1);
+        browser.keys(DELETE_KEY);
+        expect(codeInput.getInputValueByIndex(0)).toBe('');
```

Comment:
in the test name say `should return focus to the first input`, then we should add a expect here:
```expect(codeInput.getFocusedIndex()).toBe(0);```

Reply:
definitely... that was the main test for that block